dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -226,6 +228,44 @@ class ScriptsControllerTest < ActionController::TestCase
     assert_equal script, assigns(:script)
   end
 
+  test "platformization partner cannot edit our scripts" do
```

Comment:
Worth a test checking that platformization partner cannot create scripts, in light of your comment?